frb_rust/src/macros.rs

```diff
@@ -72,30 +72,35 @@ macro_rules! transfer {
 /// onmessage.forget();
 /// let port2 = channel.port2();
 /// // Declare the transferable with the same name and type
-/// flutter_rust_bridge::spawn!(|port2: MessagePort| {
+/// flutter_rust_bridge::spawn!(4, |port2: MessagePort| {
 ///     port2.post_message(&JsValue::from(true)).unwrap();
 /// });
 /// # } #[cfg(not(target_family = "wasm"))] fn main() {}
 /// ```
 #[macro_export]
 macro_rules! spawn {
-    ($($tt:tt)*) => {{
+    ($wc:expr, $($tt:tt)*) => {{
```

Comment:
uber-nit: would be great to avoid abbreviation `wc` (I guess it is worker_count?)

Reply:
Yes, it means exactly worker_count. I will fix using abbreviation :)